examples/Libevent/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../common.cmake")
+
+project(download-libevent)
+
+hunter_add_package(Libevent)
+
+find_package(Libevent CONFIG REQUIRED)
+
+add_executable(main main.cpp)
+target_link_libraries(main event)
```

Comment:
Doesn't fit the `docs/packages/pkg/Libevent.rst` snippet. Should be `Libevent::even`?

Reply:
I wish that worked. I think I have to rewrite parts of the Libevent CMake files to fix the install.